app/helpers/time_helper.rb

```diff
@@ -13,6 +13,6 @@ def timezone_select_options
   end
 
   def formatted_datetime(datetime)
-    datetime.strftime("%b %-d, %Y %l:%M %p %Z").strip
+    datetime.strftime("%b %d %l:%M %p")
```

Comment:
strip was really extraneous here... there's nothing it should ever have to strip based on this time format string.